use-cases/xtclim/README.md

```diff
@@ -24,14 +24,63 @@ The file `train.py` trains the network. Caution: It will overwrite the weights o
 
 The `anomaly.py` file evaluates the network on the available datasets - train, test, and projection.
 
-## How to launch pipeline
+## Installation
 
-The config file `pipeline.yaml` contains all the steps to execute the workflow. You can launch it from the root of the repository with:
+Please follow the documentation to install the itwinai environment.
+After that, install the required libraries within the itwinai environment with:
 
 ```bash
-python train.py -p pipeline.yaml
+pip install -r Requirements.txt
+```
+
+## How to launch pipeline locally
+
+The config file `pipeline.yaml` contains all the steps to execute the workflow. 
+This file also contains all the seasons, and a separate run is launched for each season.
+You can launch the pipeline through `train.py` from the root of the repository with:
+
+```bash
+python train.py
 
 ```
 
-## TODOs
-Integration of post-processing step + distributed strategies
+## How to launch pipeline on an HPC system
+
+The `startscript` job script can be used to launch a pipeline with SLURM on an HPC system.
+These steps should be followed to export the environment variables required by the script.
+
+```bash
+# Distributed training with torch DistributedDataParallel
+PYTHON_VENV="../../envAI_hdfml"
```

Comment:
Keep in mind that we have changed the default name of the venv to `.venv`, as this generalizes better across HPC systems. This happened when we migrated to the `uv` package manager. You are free to use different names locally, but then we suggest that you symlink it to `.venv`. Check out our document on `uv`. I believe it is called `uv-tutorial.md`.